_episodes/41-code-review.md

```diff
@@ -213,281 +153,393 @@ you will need to specify the base branch and the head branch.
-   ~~~
-   $ git clone <remote-repo-url>
-   $ cd <remote-repo-name>
-   ~~~
-   {: .language-bash}
-2. Check with the repository owner (your team member)
-   which feature (SR1.1.1 or SR1.2.1) they implemented in the
-   [previous exercise](/32-software-design/index.html#implement-requirements)
-   and what is the name of the branch they worked on.
-   Let's assume the name of the branch was `feature-x`
-   (you should amend the branch name for your case accordingly).
-3. Your task is to add tests for the code on `feature-x` branch.
-   You should do so on a separate branch called `feature-x-tests`,
-   which will branch off `feature-x`.
-   This is to enable you later on to create a pull request
-   from your `feature-x-tests` branch with your changes
-   that can then easily be reviewed and compared with `feature-x`
-   by the team member who created it.
-
-   To do so, branch off a new local branch `feature-x-tests` from the remote `feature-x` branch
-   (making sure you use the branch names that match your case).
-   Also note that, while we say "remote" branch `feature-x` -
-   you have actually obtained it locally on your machine when you cloned the remote repository.
-   ~~~
-   $ git checkout -b feature-x-tests origin/feature-x
-   ~~~
-   {: .language-bash}
-
-   You are now located in the new (local) `feature-x-tests` branch
-   and are ready to start adding your code.
-
-#### Step 3: Adding New Code
-
-> ## Exercise: Implement Tests for the New Feature
-> Look back at the
-> [solution requirements](/31-software-requirements/index.html#solution-requirements)
-> (SR1.1.1 or SR1.2.1)
-> for the feature that was implemented in your shared repository.
-> Implement tests against the appropriate specification in your local feature branch.
+Your fictional colleague has implemented it according to the specification
+and pushed it to a branch `feature-std-dev`.
+You will turn this branch into a pull request for your fictional colleague on your
+repository.
+You will then engage in code review for the change (acting as a code reviewer) on
+a course mate's repository.
+Once complete, you will respond to the pull request on your repository from another team member.
+
+### Raising a pull request for your fictional colleague
+
+1. Head over to the remote repository in GitHub
+   and locate your colleagues (`feature-std-dev`) branch from the dropdown box on the Code tab
+   (you can search for your branch or use the "View all branches" option).
+   ![All repository branches in GitHub](../fig/github-branches.png){: .image-with-shadow width="600px"}
+2. Open a pull request by clicking "Compare & pull request" button.
+   ![Submitting a pull request in GitHub](../fig/github-create-pull-request.png){: .image-with-shadow width="900px"}
+3. Select the base and the head branch, e.g. `main` and `feature-std-dev`, respectively.
+   Recall that the base branch is where you want your changes to be merged
+   and the head branch contains your changes.
```

Comment:
Make these instructions go to the pull requests page and specify the branches rather than relying on the time-sensitive button to appear.